rails-commenting.rb

```diff
@@ -33,14 +41,17 @@ def create
     end
   end
 
-  # ---6)
+# ---6)
+      # It finds a set of parametetrs that are to be edited and assigned to @post.
```

Comment:
The line referenced here declares an edit method that will provide the user with a form.